packages/contracts/controller/src/account.cairo

```diff
@@ -454,10 +504,21 @@ mod CartridgeAccount {
         fn assert_valid_span_signature(
             self: @ContractState, hash: felt252, signer_signatures: Array<SignerSignature>
         ) {
-            assert(signer_signatures.len() <= 2, 'invalid-signature-length');
+            assert(signer_signatures.len() >= 1, 'empty-signatures');
```

Comment:
```suggestion
            assert(signer_signatures.len() != 0, 'empty-signatures');
```

but i think we can do `== 2` if we always send the dummy sig

Reply:
marking as resolved since we decided against that